src/components/SlateEditor/plugins/file/Filelist.jsx

```diff
@@ -254,15 +247,20 @@ class Filelist extends React.Component {
             onClose={this.onCloseFileUploader}
             showFileUploader={showFileUploader}
           />
+          {children}
         </StyledSection>
       </Fragment>
     );
   }
 }
 
 Filelist.propTypes = {
+  attributes: PropTypes.shape({
+    'data-key': PropTypes.string.isRequired,
+  }),
```

Comment:
Tror denne kan settes til å være av typen object. data-key stemte for gamle slate-versjonen, men attributes innholder nå noe annet nå. 

Kan alternativt skrive om filen til typescript. (se for eksempel plugins/concept)
```
interface Props {
  ...
  attributes: RenderElementProps['attributes'];
  ...
}
```